src/app/handbook/page.js

```diff
@@ -0,0 +1,199 @@
+import Title from "@/components/Title";
+import Subtitle from "@/components/handbook/Subtitle";
+
+const Handbook = () => {
+  return (
+    <div className="flex flex-col items-center justify-center p-[4%]">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/5c264694-2a6a-4508-ba0b-f2a1b7ec34b6)

- make font size of body text a little bigger